test/e2e/demos_test.go

```diff
@@ -68,6 +73,10 @@ var _ = Describe("Introduction", func() {
 	})
 
 	Context("All tutorials in a row", func() {
+		AfterEach(func() {
```

Comment:
btw you can do AfterEach(cleanConfiguration)

Reply:
Right!